test/regressions/xhr-response-xml-body.test.ts

```diff
@@ -0,0 +1,67 @@
+import { createInterceptor } from '../../src';
+import { interceptXMLHttpRequest } from '../../src/interceptors/XMLHttpRequest';
+import { createXMLHttpRequest } from '../helpers';
+
+const XML_STRING = '<node key="value">Content</node>'
+
+describe('Content-Type: application/xml', () => {
```

Comment:
I wonder if there is a better way to consolidate these tests while still checking that both `Content-Type` headers will yield a value for `responseXML`. 🤔 As far as I can tell, `text/xml` and `application/xml` are used fairly interchangeably, but [RFC (3023)](https://tools.ietf.org/html/rfc3023) specifies the following:
> If an XML document -- that is, the unprocessed, source XML document -- is readable by casual users, text/xml is preferable to application/xml.  MIME user agents (and web user agents) that do not have explicit support for text/xml will treat it as text/plain, for example, by displaying the XML MIME entity as plain text. Application/xml is preferable when the XML MIME entity is unreadable by casual users.